k8s/upgrades/0.8.0-0.8.1/cstor-target-patch.tpl.json

```diff
@@ -0,0 +1,16 @@
+{
+    "metadata": {
+       "annotations": {
+          "openebs.io/storage-class-ref": "name: @sc_name\nresourceVersion: @sc_r_ver\n"
```

Comment:
can we have better/understood names. e.g. @sc_r_ver is not well understood

Reply:
@ashishranjan738 have you tested this for any json format & indentation issues?
e.g. Do we not require commas & double quotes?

Reply:
Can you write UT in .go to test these files?
e.g. k8s/upgrades/tests/0.8.0-0.8.1/ or something!!!

Reply:
will try to write UT for this

Reply:
@AmitKumarDas the json seems to work fine.